unit_tests/fields/multi_field_test.cpp

```diff
@@ -22,15 +22,14 @@ FLOW123D_FORCE_LINK_IN_PARENT(field_constant)
 string field_constant_input = R"JSON(
 {
     common={ 
-        TYPE="MultiField", 
-        component_names=[ "field_1", "field_2", "field_3" ],
-        common={ TYPE="FieldConstant", value=[1, 2, 3]}
+        TYPE="FieldConstant", 
+        value=[1, 2, 3]
     },
-    transposed={ 
-        TYPE="MultiField", 
-        component_names=[ "field_1", "field_2", "field_3" ],
-        components=[ { TYPE="FieldConstant", value=1}, { TYPE="FieldConstant", value=2}, { TYPE="FieldConstant", value=3} ]
-    }
+    transposed=[ 
+        { TYPE="FieldConstant", value=1},
+        { TYPE="FieldConstant", value=2},
+        { TYPE="FieldConstant", value=3}
```

Comment:
Test also multifield with components given by different algorithms.
Do we have test of multifield with components of type FieldElementwise? There are some tricks in GMSH reader that needs testing.